### PR TITLE
All cloudscale.ch modules are moved to a collection

### PR DESCRIPTION
**Pull request by gaudenz**
##### SUMMARY
The cloudscale.ch modules and plugins were moved from the community collection to their own collection.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
cloudscale_server
cloudscale_floating_ip
cloudscale_volume
cloudscale_server_group
cloudscale inventory plugin

### CODE REVIEW

File: lib/ansible/config/ansible_builtin_runtime.yml [hunk 1]
```diff
@@ -8313,7 +8313,7 @@ plugin_routing:
     ce:
       redirect: community.network.ce
     cloudscale:
-      redirect: community.general.cloudscale
+      redirect: cloudscale_ch.cloud.doc_fragments
```
Comment (gundalow) [inline lib/ansible/config/ansible_builtin_runtime.yml]: This should be name of the file under `plugins/doc_fragment/`
```suggestion
      redirect: cloudscale_ch.cloud.api_parameters
```

Simialr for module_utils & inventory

Comment (gaudenz) [reply lib/ansible/config/ansible_builtin_runtime.yml]: Fixed, AFAICS the path for the inventory plugin was already correct.

**Comment by felixfontein** [comment]
You also need to update .github/BOTMETA.yml (https://github.com/ansible/ansible/blob/792755c5a3cf7b38dcafc27e1dbdefe66f12966b/.github/BOTMETA.yml).

**Review by resmo — APPROVED** [review]
LGTM 

(see https://github.com/cloudscale-ch/ansible-collection-cloudscale/tree/master/plugins/inventory)

**Review by felixfontein — APPROVED** [review]

**Review by gundalow — APPROVED** [review]

**Pull request merged**